### PR TITLE
Optimized home page's A11y and seo scores

### PR DESCRIPTION
**Pull request by itaim18**
solves #83

### CODE REVIEW

File: components/Common/Darkmode.tsx [hunk 1]
```diff
@@ -9,6 +9,8 @@ const Darkmode: React.FC = () => {
 
   return (
     <button
+      id="theme-toggler"
+      aria-label="click here to toggle theme"
```
Comment (adirkandel) [inline components/Common/Darkmode.tsx:13]: Please add also the theme name like -> 
toggle theme to dark/light them

File: components/Footer/FacebookIconLink.tsx [hunk 2]
```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 const FacebookIconLink = () => (
-  <a href="https://www.facebook.com/" target="_blank">
+  <a
+    href="https://www.facebook.com/"
+    target="_blank"
+    aria-label="See us in facebook"
```
Comment (adirkandel) [inline components/Footer/FacebookIconLink.tsx:6]: Please change it to hebrew

File: components/Footer/LinkedinIconLink.tsx [hunk 3]
```diff
@@ -2,7 +2,11 @@ import React from 'react';
 
 const LinkedinIconLink = () => {
   return (
-    <a href="https://www.linkedin.com/" target="_blank">
+    <a
+      href="https://www.linkedin.com/"
+      target="_blank"
+      aria-label="See us in linkedin"
```
Comment (adirkandel) [inline components/Footer/LinkedinIconLink.tsx:8]: ditto

File: components/Footer/TwitterIconLink.tsx [hunk 4]
```diff
@@ -2,7 +2,11 @@ import React from 'react';
 
 const TwitterIconLink = () => {
   return (
-    <a href="https://twitter.com/" target="_blank">
+    <a
+      href="https://twitter.com/"
+      target="_blank"
+      aria-label="See us in twitter"
```
Comment (adirkandel) [inline components/Footer/TwitterIconLink.tsx:8]: ditto

**Review by RoyTennenbaum — APPROVED** [review]
lgtm

love the placeholder optimization 💯

**Pull request merged**